src/AppInstallerCommonCore/Manifest/ManifestCommon.cpp

```diff
@@ -614,6 +633,28 @@ namespace AppInstaller::Manifest
                 { HRESULT_FROM_WIN32(ERROR_INSTALL_CANCEL), ExpectedReturnCodeEnum::CancelledByUser },
                 { HRESULT_FROM_WIN32(ERROR_PACKAGE_ALREADY_EXISTS), ExpectedReturnCodeEnum::AlreadyInstalled },
                 { HRESULT_FROM_WIN32(ERROR_INSTALL_PACKAGE_DOWNGRADE), ExpectedReturnCodeEnum::Downgrade },
+                { HRESULT_FROM_WIN32(ERROR_DEPLOYMENT_BLOCKED_BY_POLICY), ExpectedReturnCodeEnum::BlockedByPolicy},
+                { HRESULT_FROM_WIN32(ERROR_INSTALL_POLICY_FAILURE), ExpectedReturnCodeEnum::BlockedByPolicy},
+                { HRESULT_FROM_WIN32(ERROR_PACKAGES_IN_USE), ExpectedReturnCodeEnum::PackageInUse },
+                /*
+                * For the below return codes, 'argument': conversion from 'HRESULT' to 'unsigned long', signed/unsigned mismatch
+                * 
+                { HRESULT_FROM_WIN32(APPX_E_INVALID_PACKAGING_LAYOUT), ExpectedReturnCodeEnum::InvalidParameter },

```

Comment:
These appx errors are already HRESULT, so no need to use HRESULT_FROM_WIN32.

But I feel these appx errors do not belong to InvalidParameter category perfectly, as they are corrupted appx/msix package.

For example, I doubt APPX_E_INVALID_PACKAGESIGNCONFIG will ever happen during deployment because this is a package signing time config file, we should probably remove these error code mapping for all the commented out ones